src/main/java/duke/taskOperations/StringToRemoveFormat.java

```diff
@@ -0,0 +1,51 @@
+package duke.taskOperations;
+
+public class StringToRemoveFormat {
+    public static String getStringToRemove(String typeOfTask, String taskIsDone, String rawTaskDescription) {
+        String textToRemove;
+        if (typeOfTask.equals("T") == true) {
+            textToRemove = getStringToDoFormat(taskIsDone, rawTaskDescription);
+        } else if (typeOfTask.equals("E") == true) {
```

Comment:
"T" and "E" are quite arbitrary values to check - giving more meaningful names will give readers a better overview of what `getStringToRemove` entails.